lib/web_ui/lib/src/engine/html/picture.dart

```diff
@@ -51,7 +51,11 @@ class _PaintRequest {
 List<_PaintRequest> _paintQueue = <_PaintRequest>[];
 
 void _recycleCanvas(EngineCanvas? canvas) {
-  assert(canvas == null || !_recycledCanvases.contains(canvas));
+  /// If a canvas is in the paint queue it maybe be recycled. To
```

Comment:
very nit: // comments inside the methods

Reply:
Done.